pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -152,14 +147,20 @@ func (ipa *InterPodAffinity) CalculateInterPodAffinityPriority(pod *v1.Pod, node
 			// increment <pm.counts> for every node in the cluster with the same <term.TopologyKey>
 			// value as that of <existingPods>`s node by the term`s weight.
 			terms := affinity.PodAffinity.PreferredDuringSchedulingIgnoredDuringExecution
-			pm.processTerms(terms, pod, existingPod, existingPodNode, 1)
+			err := pm.processTerms(terms, pod, existingPod, existingPodNode, 1)
+			if err != nil {
+				return err
+			}
```

Comment:
you can do 

```
if err := pm.processTerms(terms, pod, existingPod, existingPodNode, 1); err != nil {
    return err
}
```
here and all other places below.

Reply:
Not sure which style we prefer. I have seen many places that the current pattern is used. But I will go ahead with you suggestion since I like this style too.